CHANGELOG.md

```diff
@@ -6,7 +6,20 @@ This change log file is based on best practices from [Keep a Changelog](http://k
 This project adheres to [Semantic Versioning](http://semver.org/). Breaking changes result in a different MAJOR version. UI changes that might break customizations on top of the SDK will be treated as breaking changes too.
 This project adheres to the Node [default version scheme](https://docs.npmjs.com/misc/semver).
 
-## [next-version]
+## [6.12.0] - 2021-08-09
```

Comment:
Need to update the date since we're aiming to release on 10 Aug now.

Reply:
Updated